### ISSUE TITLE
Stryker 0.5.0 crashes on simple project

### ISSUE THREAD

**Issue by richardwerkman** [stryker-mutator/stryker-net#234]
Stryker crashes on a simple project. It seems to fail inside the `TotalNumberOfTestsParser`.

```
[22:07:26 INF] Using C:\Dev\ExampleProjects\ExampleProject\ExampleProject.XUnit\ExampleProject.XUnit.csproj as project file
[22:07:26 INF] Starting initial build
[22:07:28 INF] Initial build successful
[22:07:29 INF] Using testrunner DotnetTestRunner
[22:07:29 INF] Initial testrun started
[22:07:31 ERR] An error occurred during the mutation test run
System.IO.FileNotFoundException: Could not load file or assembly 'System.Memory, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The system cannot find the file specified.
File name: 'System.Memory, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'
   at Stryker.Core.Parsers.TotalNumberOfTestsParser.ParseTotalNumberOfTests(String testProcessRunnerOutput)
   at Stryker.Core.TestRunners.DotnetTestRunner.RunAll(Nullable`1 timeoutMS, Nullable`1 activeMutationId)
   at Stryker.Core.Initialisation.InitialTestProcess.InitialTest(ITestRunner testRunner)
   at Stryker.Core.Initialisation.InitialisationProcess.Initialize(StrykerOptions options)
   at Stryker.Core.StrykerRunner.RunMutationTest(StrykerOptions options)

[22:07:31 INF] Time Elapsed 00:00:05.2187106
```

**Comment by chris579** [comment]
Can confirm this bug for .net core 2.1 with NUnit. Installing the latest `System.Memory` package does not resolve this issue.

**Comment by rvanmaanen** [comment]
Same issue here. Tried with .NET Core SDK 2.1.403 and 2.2.100-preview3-009430. Code compiles and tests all run on my machine. Also tried installing System.Memory.

**Comment by richardwerkman** [comment]
I can't reproduce this issue locally when running in dev mode. The issue only occurs when running directly from the nuget package. I updated the target framework to netcoreapp2.1 instead of 2.0 and created a local nuget package. The issue seems to be resolved that way.

We will update our integration test to actually install Stryker from a local NuGet feed to reproduce this issue in the build. That way we hope to prevent these kind of issues to occur in the future.

**Comment by vinnytheviking** [comment]
Good Morning

Any idea when this fix will be pushed as part of a release ?

I'm trying to introduce Stryker into my org as a pre-requisite for testing.

Thanks

Vinny

**Comment by Mobrockers** [comment]
A new release with this fix will go out soon

**Comment by Mobrockers** [comment]
@vinnytheviking the release should be available on nuget.org within 15 minutes. The packages have been released with version number 0.6.0.

**Comment by vinnytheviking** [comment]
Thank you for the quick turn-around.

much apprecaited
